theme/tutorial.less

```diff
@@ -274,6 +274,11 @@ code.lang-filterblocks {
     }
 }
 
+#tutorialcard .isValidated {
+    background: @green;
```

Comment:
nit: probably define a less variable to allow changing theming the "validated" color